File: Noodum.Wokamon.Sunny/Noodum.Wokamon.Sunny.SensorDataCollector/Models/PhoneData.cs
using Noodum.Wokamon.Sunny.Core.Models;

namespace Noodum.Wokamon.Sunny.SensorDataCollector.Models
{
    public class PhoneData
    {
        public PhoneType PhoneType { get; set; }
        public PhoneState PhoneState { get; set; }
        public int UpdateInterval { get; set; }
    }
}

File: MovingDataCollector.iOS/MovingDataCollector/Models/PhoneData.m
//
//  PhoneData.m
//  MovingDataCollector
//
//  Created by Jianming Xiao on 7/24/14.
//  Copyright (c) 2014 jianming.xiao. All rights reserved.
//

#import "PhoneData.h"

@implementation PhoneData

- (NSDictionary *)dictionary {
    return @{@"PhoneType" : @(self.phoneType),
            @"PhoneState" : @(self.phoneState),
            @"UpdateInterval" : @(self.updateInterval)};
}

@end
